lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -10175,22 +10175,8 @@ namespace Js
 
     BOOL JavascriptOperators::IsIterable(RecyclableObject* instance, ScriptContext* scriptContext)
     {
-        if (JavascriptProxy::Is(instance))
-        {
-            Var func = JavascriptOperators::GetProperty(instance, PropertyIds::_symbolIterator, scriptContext);
-            if (JavascriptOperators::IsUndefinedObject(func))
-            {
-                return FALSE;
-            }
-            else
-            {
-                return TRUE;
-            }
-        }
-        else
-        {
-            return JavascriptOperators::HasProperty(instance, PropertyIds::_symbolIterator);
-        }
+        Var func = JavascriptOperators::GetProperty(instance, PropertyIds::_symbolIterator, scriptContext);
```

Comment:
Sorry, I didn't see that issue. I pushed a new commit which fixes both #667 and #584